install only the given value in LINGUAS
signed-of: Joerg Bornkessel <hd_brummy@gentoo.org> ( 2012/04/15 )

diff -Naur vdr-1.7.27.orig/Makefile vdr-1.7.27/Makefile
--- vdr-1.7.27.orig/Makefile	2012-04-15 15:45:18.000000000 +0200
+++ vdr-1.7.27/Makefile	2012-04-15 15:46:21.000000000 +0200
@@ -144,7 +144,7 @@
 
 PODIR     = po
 LOCALEDIR = locale
-I18Npo    = $(wildcard $(PODIR)/*.po)
+I18Npo    = $(foreach dir,$(LINGUAS),$(wildcard $(PODIR)/$(dir)*.po))
 I18Nmsgs  = $(addprefix $(LOCALEDIR)/, $(addsuffix /LC_MESSAGES/vdr.mo, $(notdir $(foreach file, $(I18Npo), $(basename $(file))))))
 I18Npot   = $(PODIR)/vdr.pot
 
@@ -167,7 +167,7 @@
 
 install-i18n:
 	@mkdir -p $(DESTDIR)$(LOCDIR)
-	@(cd $(LOCALEDIR); cp -r --parents * $(DESTDIR)$(LOCDIR))
+	@(cd $(LOCALEDIR); for linguas in $(LINGUAS); do [ "$$linguas" = "en" ] && continue; cp -r --parents $$linguas* $(DESTDIR)$(LOCDIR); done)
 
 # The 'include' directory (for plugins):
 
